CHANGELOG.md

```diff
@@ -1,3 +1,27 @@
+## Supported Release 2.3.0
+### Summary
+A release containing many bugfixes with additional features, including a modulesync update.
+
+#### Features
+- Apt_updates facts now use /usr/bin/apt-get.
+- Addition of notify update to apt::source.
+- Update to newest modulesync_configs.
+- Installs software-properties-common for xenial.
+- Modulesync updates.
+- Add ability to specify a hash of apt::conf defines.
+
+#### Bugfixes
+- Update ke_compat_spec to run under STRICT_VARIABLES.
+- A clean up of key_compat_specs.
+- Apt::setting expects priority to be an integer, set defaults accordingly.
+- Fixed version check for Ubuntu on 16.04.
+- Now uses hkps.pool.sks-keyservers.net instead of pgp.mit.edu.
+- Updates and fixes to tests. General cleanup.
+- Fixed regexp for $ensure params.
+- Apt/params: Remove unused LSB facts.
+- Replaced `-s` with `-f` in ppa rspec tests.
```

Comment:
Maybe we should explain what this does? Just a flag change is not very informative.